tests/Unit/Products/UnitsModelTest.php

```
@@ -51,21 +49,19 @@ public function it_returns_empty_collection_when_no_units_exist(): void
+        $this->assertFalse($this->model->exists('NonExistent'));
     }
 
     #[Test]
     public function it_returns_singular_name_for_quantity_of_one(): void
     {
         /* Arrange */
-        $unit = Unit::create(['unit_name' => 'Hour', 'unit_name_plrl' => 'Hours']);
 
         /* Act */
-        $result = $this->service->getName($unit->unit_id, 1);
+        $result = $this->model->getName($unit->unit_id, 1);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$unit'. (undefined)

(UnusedLocalVariable)